CHANGELOG.md

```diff
@@ -6,6 +6,50 @@
 
 **Added**:
 
+- **decidim-assemblies**: Add feature filter assemblies by type [\#4659](https://github.com/decidim/decidim/pull/4659/)
```

Comment:
Can you fix this, please? Seems like it's trying to add changelogs from previous releases

Reply:
Done!